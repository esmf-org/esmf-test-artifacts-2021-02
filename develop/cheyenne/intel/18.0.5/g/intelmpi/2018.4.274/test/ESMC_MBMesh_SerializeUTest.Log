20201117 075609.121 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20201117 075609.121 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20201117 075609.121 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20201117 075609.121 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20201117 075609.121 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20201117 075609.121 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20201117 075609.121 INFO             PET0 Running with ESMF Version   : ESMF_8_1_0_beta_snapshot_37-1-g18a355632c
20201117 075609.121 INFO             PET0 ESMF library build date/time: "Nov 17 2020" "07:03:52"
20201117 075609.121 INFO             PET0 ESMF library build location : /glade/scratch/himanshu/scripts_dir/daily/intel_cheyenne/esmf
20201117 075609.121 INFO             PET0 ESMF_COMM                   : intelmpi
20201117 075609.121 INFO             PET0 ESMF_NETCDF                 : disabled
20201117 075609.121 INFO             PET0 ESMF_PNETCDF                : disabled
20201117 075609.121 INFO             PET0 ESMF_PIO                    : enabled
20201117 075609.121 INFO             PET0 ESMF_YAMLCPP                : enabled
20201117 075609.121 INFO             PET0 ESMF_MOAB                   : enabled
20201117 075609.121 INFO             PET0 Beginning Test, file ESMC_MBMesh_SerializeUTest.C, line 104
20201117 075609.121 INFO             PET0 NUMBER_OF_PROCESSORS 4
20201117 075609.122 INFO             PET0 PASS Serialization of an MBMesh, ESMC_MBMesh_SerializeUTest.C, line 149
20201117 075609.122 INFO             PET0 PASS Deserialization of an MBMesh, ESMC_MBMesh_SerializeUTest.C, line 164
20201117 075609.123 INFO             PET0 Ending Test, file ESMC_MBMesh_SerializeUTest.C, line 179
20201117 075609.123 INFO             PET0 Finalizing ESMF
20201117 075609.121 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20201117 075609.121 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20201117 075609.121 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20201117 075609.121 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20201117 075609.121 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20201117 075609.121 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20201117 075609.121 INFO             PET1 Running with ESMF Version   : ESMF_8_1_0_beta_snapshot_37-1-g18a355632c
20201117 075609.121 INFO             PET1 ESMF library build date/time: "Nov 17 2020" "07:03:52"
20201117 075609.121 INFO             PET1 ESMF library build location : /glade/scratch/himanshu/scripts_dir/daily/intel_cheyenne/esmf
20201117 075609.121 INFO             PET1 ESMF_COMM                   : intelmpi
20201117 075609.121 INFO             PET1 ESMF_NETCDF                 : disabled
20201117 075609.121 INFO             PET1 ESMF_PNETCDF                : disabled
20201117 075609.121 INFO             PET1 ESMF_PIO                    : enabled
20201117 075609.121 INFO             PET1 ESMF_YAMLCPP                : enabled
20201117 075609.121 INFO             PET1 ESMF_MOAB                   : enabled
20201117 075609.121 INFO             PET1 Beginning Test, file ESMC_MBMesh_SerializeUTest.C, line 104
20201117 075609.121 INFO             PET1 NUMBER_OF_PROCESSORS 4
20201117 075609.122 INFO             PET1 PASS Serialization of an MBMesh, ESMC_MBMesh_SerializeUTest.C, line 149
20201117 075609.122 INFO             PET1 PASS Deserialization of an MBMesh, ESMC_MBMesh_SerializeUTest.C, line 164
20201117 075609.123 INFO             PET1 Ending Test, file ESMC_MBMesh_SerializeUTest.C, line 179
20201117 075609.123 INFO             PET1 Finalizing ESMF
20201117 075609.121 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20201117 075609.121 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20201117 075609.121 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20201117 075609.121 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20201117 075609.121 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20201117 075609.121 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20201117 075609.121 INFO             PET2 Running with ESMF Version   : ESMF_8_1_0_beta_snapshot_37-1-g18a355632c
20201117 075609.121 INFO             PET2 ESMF library build date/time: "Nov 17 2020" "07:03:52"
20201117 075609.121 INFO             PET2 ESMF library build location : /glade/scratch/himanshu/scripts_dir/daily/intel_cheyenne/esmf
20201117 075609.121 INFO             PET2 ESMF_COMM                   : intelmpi
20201117 075609.121 INFO             PET2 ESMF_NETCDF                 : disabled
20201117 075609.121 INFO             PET2 ESMF_PNETCDF                : disabled
20201117 075609.121 INFO             PET2 ESMF_PIO                    : enabled
20201117 075609.121 INFO             PET2 ESMF_YAMLCPP                : enabled
20201117 075609.121 INFO             PET2 ESMF_MOAB                   : enabled
20201117 075609.121 INFO             PET2 Beginning Test, file ESMC_MBMesh_SerializeUTest.C, line 104
20201117 075609.121 INFO             PET2 NUMBER_OF_PROCESSORS 4
20201117 075609.122 INFO             PET2 PASS Serialization of an MBMesh, ESMC_MBMesh_SerializeUTest.C, line 149
20201117 075609.122 INFO             PET2 PASS Deserialization of an MBMesh, ESMC_MBMesh_SerializeUTest.C, line 164
20201117 075609.123 INFO             PET2 Ending Test, file ESMC_MBMesh_SerializeUTest.C, line 179
20201117 075609.123 INFO             PET2 Finalizing ESMF
20201117 075609.121 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20201117 075609.121 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20201117 075609.121 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20201117 075609.121 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20201117 075609.121 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20201117 075609.121 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20201117 075609.121 INFO             PET3 Running with ESMF Version   : ESMF_8_1_0_beta_snapshot_37-1-g18a355632c
20201117 075609.121 INFO             PET3 ESMF library build date/time: "Nov 17 2020" "07:03:52"
20201117 075609.121 INFO             PET3 ESMF library build location : /glade/scratch/himanshu/scripts_dir/daily/intel_cheyenne/esmf
20201117 075609.121 INFO             PET3 ESMF_COMM                   : intelmpi
20201117 075609.121 INFO             PET3 ESMF_NETCDF                 : disabled
20201117 075609.121 INFO             PET3 ESMF_PNETCDF                : disabled
20201117 075609.121 INFO             PET3 ESMF_PIO                    : enabled
20201117 075609.121 INFO             PET3 ESMF_YAMLCPP                : enabled
20201117 075609.121 INFO             PET3 ESMF_MOAB                   : enabled
20201117 075609.121 INFO             PET3 Beginning Test, file ESMC_MBMesh_SerializeUTest.C, line 104
20201117 075609.121 INFO             PET3 NUMBER_OF_PROCESSORS 4
20201117 075609.122 INFO             PET3 PASS Serialization of an MBMesh, ESMC_MBMesh_SerializeUTest.C, line 149
20201117 075609.122 INFO             PET3 PASS Deserialization of an MBMesh, ESMC_MBMesh_SerializeUTest.C, line 164
20201117 075609.123 INFO             PET3 Ending Test, file ESMC_MBMesh_SerializeUTest.C, line 179
20201117 075609.123 INFO             PET3 Finalizing ESMF
Tue Nov 17 10:06:45 MST 2020
