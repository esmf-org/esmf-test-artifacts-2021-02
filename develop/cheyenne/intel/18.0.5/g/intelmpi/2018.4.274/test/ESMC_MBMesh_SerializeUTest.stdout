Beginning Test, file ESMC_MBMesh_SerializeUTest.C, line 103
NUMBER_OF_PROCESSORS 4
Beginning Test, file ESMC_MBMesh_SerializeUTest.C, line 103
NUMBER_OF_PROCESSORS 4
Beginning Test, file ESMC_MBMesh_SerializeUTest.C, line 103
NUMBER_OF_PROCESSORS 4
Beginning Test, file ESMC_MBMesh_SerializeUTest.C, line 103
NUMBER_OF_PROCESSORS 4
PASS Serialization of an MBMesh, ESMC_MBMesh_SerializeUTest.C, line 145
PASS Serialization of an MBMesh, ESMC_MBMesh_SerializeUTest.C, line 145
PASS Serialization of an MBMesh, ESMC_MBMesh_SerializeUTest.C, line 145
MESH PET 0 - size = 4
 * MESH PET 1 - size = 4
 * MESH PET 2 - size = 4
 * MESH PET 3 - size = 4
 * PASS Serialization of an MBMesh, ESMC_MBMesh_SerializeUTest.C, line 145
PET PET PET 1 - Element 12 - Element 1
PET 3 - Element 1
 * 0 - Element 1
 * PET 0 - Element 2
 * 
 * PET 1 - Element 2
 * PET 1 - Element 3
 * PET 1 - Element 4
 * PET 2 - Element 2
 * PET 2 - Element 3
 * PET 2 - Element 4
PET 3 - Element 2
 * PET 3 - Element 3
 * PET 3 - Element 4
PET 0 - Element 3
 * PET 0 - Element 4
PASS Deserialization of an MBMesh, ESMC_MBMesh_SerializeUTest.C, line 161
PASS Deserialization of an MBMesh, ESMC_MBMesh_SerializeUTest.C, line 161
MESH PET 0 - size = 0
MESH PET 1 - size = 0
MESH PET 2 - size = 0
MESH PET 3 - size = 0
PASS Deserialization of an MBMesh, ESMC_MBMesh_SerializeUTest.C, line 161
PASS Deserialization of an MBMesh, ESMC_MBMesh_SerializeUTest.C, line 161
Ending Test, file ESMC_MBMesh_SerializeUTest.C, line 173
Ending Test, file ESMC_MBMesh_SerializeUTest.C, line 173
Ending Test, file ESMC_MBMesh_SerializeUTest.C, line 173
Ending Test, file ESMC_MBMesh_SerializeUTest.C, line 173
 PET 2 Test Elapsed Time  2.231000 msec.
 PET 1 Test Elapsed Time  2.147000 msec.
 PET 3 Test Elapsed Time  2.346000 msec.
 PET 0 Test Elapsed Time  2.204000 msec.
Fri Dec  4 16:41:09 MST 2020
